packages/trie/README.md

```diff
@@ -6,23 +6,27 @@
 [![Code Coverage][trie-coverage-badge]][trie-coverage-link]
 [![Discord][discord-badge]][discord-link]
 
-This is an implementation of the modified merkle patricia tree as specified in the [Ethereum Yellow Paper](http://gavwood.com/Paper.pdf):
+This is an implementation of the [Modified Merkle Patricia Trie](https://ethereum.org/en/developers/docs/data-structures-and-encoding/patricia-merkle-trie/) as specified in the [Ethereum Yellow Paper](http://gavwood.com/Paper.pdf):
 
-> The modified Merkle Patricia tree (trie) provides a persistent data structure to map between arbitrary-length binary data (byte arrays). It is defined in terms of a mutable data structure to map between 256-bit binary fragments and arbitrary-length binary data. The core of the trie, and its sole requirement in terms of the protocol specification is to provide a single 32-byte value that identifies a given set of key-value pairs.
+> The modified Merkle Patricia tree (trie) provides a persistent data structure to map between arbitrary-length binary data (byte arrays). It is defined in terms of a mutable data structure to map between 256-bit binary fragments and arbitrary-length binary data. The core of the trie, and its sole requirement in terms of the protocol specification, is to provide a single 32-byte value that identifies a given set of key-value pairs.
 
-The only backing store supported is LevelDB through the `levelup` module.
+## Installation
 
-# INSTALL
```

Comment:
This is the header naming scheme in (more or less) all our other packages, and I wouldn't want to change this for one package here along the sidelines.

Reply:
So this would be for all before-level-1-after-PR-level2 headlines I guess. Also this level changing get's really rather even more inconsistent if we only change on one package.

So I am open to whatever formatting you come up with and things like capitalization or not or whatever, but we should do across the reops.

Reply:
(if you want to do I would be also fine to do this within this PR)